guides/v2.2/release-notes/ReleaseNotes2.2.0CE.md

```diff
@@ -339,7 +339,7 @@ This release contains hundreds of fixes and enhancements.
 <!---70256 -->* You can now create an `etc/view.xml` file containing an `images` tag with an attribute `module`. *Fix submitted by community member [Marius Strajeru](https://github.com/tzyganu){: target="_blank"} in pull request [10052](https://github.com/magento/magento2/pull/10052){: target="_blank"}.*
 
 
-<!--- 70345-->* Magento now displays the Category selection UI under Conditions when you select a rule for editing. *Fix submitted by community member [duckchip](https://github.com/duckchip){: target="_blank"} in pull request [10094](https://github.com/magento/magento2/pull/10094){: target="_blank"}.*
```

Comment:
I started taking random deleted links here, because I thought it odd that we'd delete links to user profiles. This one works, as does the next one (for user `barbazul`). The one below that (`ajpevers`) doesn't work. We're obviously not deleting all user profile links, so something is wrong with the link validator and/or the regex used to strip out the links.

Reply:
I deleted all the links to user profiles to avoid broken builds each time someone removed their profile.

Reply:
If all the links to user profiles were deleted, I wouldn't have commented. The number of remaining links is greater than deleted links. As it stands right now, it looks totally random.

Reply:
I see now, I expected that both the user name and the link contain the same [duckchip](https://github.com/duckchip) in a link and in a text. But this was not true for all cases.

Reply:
Removed more of them.